### PR TITLE
[FIX][mass_editing] v11 Fixing error when removing value for fields b…

### PR DESCRIPTION
**Pull request by BT-rmartin**
…eing translatable, as their translations were not removed.

### CODE REVIEW

**Comment by BT-rmartin** [comment]
@pedrobaeza Hi Pedro, could you please have a look at this? It's exactly the same functionality as the one you merged right now. Thanks

File: mass_editing/models/mass_object.py [hunk 1]
```diff
@@ -77,6 +77,7 @@ def unlink(self):
         self.unlink_action()
         return super(MassObject, self).unlink()
 
+    @api.multi
```
Comment (pedrobaeza) [inline mass_editing/models/mass_object.py]: No need for this. It's the default

File: mass_editing/__manifest__.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
     'version': '11.0.1.0.0',
     'author': 'Serpent Consulting Services Pvt. Ltd., '
               'Tecnativa, '
+              'brain-tec AG, '
```
Comment (pedrobaeza) [inline mass_editing/__manifest__.py]: For this newest version, I don't consider your contribution to be upgraded to co-authorship. Please keep only the contributor in the README part.

File: mass_editing/tests/test_mass_editing.py [hunk 3]
```diff
@@ -16,6 +16,9 @@ def setUp(self):
         self.mass_wiz_obj = self.env['mass.editing.wizard']
         self.mass_object_model = self.env['mass.object']
         self.res_partner_model = self.env['res.partner']
+        self.ir_translation_model = self.env['ir.translation']
+        self.lang_model = self.env['res.lang']
+
```
Comment (pedrobaeza) [inline mass_editing/tests/test_mass_editing.py]: Don't add empty extra lines in the middle of a method.

File: mass_editing/tests/test_mass_editing.py [hunk 4]
```diff
@@ -38,33 +53,52 @@ def _create_partner(self):
             'email': 'example@yourcompany.com',
             'phone': 123456,
             'category_id': [(6, 0, categ_ids)],
+            'notify_email': 'always'
+        })
+
+    def _create_partner_title(self):
+        """Create a Partner Title."""
+
```
Comment (pedrobaeza) [inline mass_editing/tests/test_mass_editing.py]: Remove all the empty lines

File: mass_editing/tests/test_mass_editing.py [hunk 5]
```diff
@@ -38,33 +53,52 @@ def _create_partner(self):
...
             'email': 'test@test.com',
         })
 
-    def _create_mass_editing(self, model, fields):
+    def _create_mass_editing(self, model, fields, model_name):
         """Create a Mass Editing with Partner as model and
         email field of partner."""
         mass = self.mass_object_model.create({
-            'name': 'Mass Editing for Partner',
+            'name': u'Mass Editing for {0}'.format(model_name),
```
Comment (pedrobaeza) [inline mass_editing/tests/test_mass_editing.py]: In Python 3, there's no more `u''`, as everything is unicode.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: mass_editing/models/mass_object.py (hunk 1 above)
Comment (BT-rmartin) [reply mass_editing/models/mass_object.py]: Travis was complaining about it was missing, at least in previous versions

File: mass_editing/tests/test_mass_editing.py (hunk 3 above)
Comment (BT-rmartin) [reply mass_editing/tests/test_mass_editing.py]: Why not? We separate model connections, to the logic itself. IMHO it improves legibility.

File: mass_editing/tests/test_mass_editing.py (hunk 4 above)
Comment (BT-rmartin) [reply mass_editing/tests/test_mass_editing.py]: Again, I'm for them, to improve legibility. Any reason behind this suggestion?

File: mass_editing/tests/test_mass_editing.py (hunk 5 above)
Comment (BT-rmartin) [reply mass_editing/tests/test_mass_editing.py]: Ok, thanks

File: mass_editing/__manifest__.py (hunk 2 above)
Comment (BT-rmartin) [reply mass_editing/__manifest__.py]: Ok, I understand this. Thanks for the suggestion

File: mass_editing/tests/test_mass_editing.py (hunk 3 above)
Comment (pedrobaeza) [reply mass_editing/tests/test_mass_editing.py]: No, it only adds unneeded lines. PEP8 recommends not to put them. If you want to add a line with a comment, OK. That's the way to separate things inside the same method. Adding empty lines, not.

File: mass_editing/models/mass_object.py (hunk 1 above)
Comment (pedrobaeza) [reply mass_editing/models/mass_object.py]: I'm sure it doesn't.

File: mass_editing/tests/test_mass_editing.py (hunk 3 above)
Comment (BT-rmartin) [reply mass_editing/tests/test_mass_editing.py]: I already removed the empty lines. Thanks for the suggestion

**Review by pedrobaeza — APPROVED** [review]
Fast-tracking as it has been already merged in v10.

**Pull request merged**